.gitignore

```diff
@@ -10,6 +10,7 @@ ocr/weather/*
 configs/config.ini
 *.calc
 configs/geofences/*
+configs/confbak/*
```

Comment:
I guess this is something you needed? Not for the main repo?

Reply:
Yeah my mistake. Gonna remove it. Sorry